doc/api/assert.md

```diff
@@ -322,6 +322,73 @@ function func() {}
 const callsfunc = tracker.calls(func);
 ```
 
+### `tracker.callsWith([fn],withArgs[, exact])`
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+* `fn` {Function} **Default:** A no-op function.
+* `withArgs` {Array} **Default:** An array with the arguments list.
+* `exact` {number} **Default:** `1`.
+* Returns: {Function} that wraps `fn`.
+
+The wrapper function is expected to be called exactly `exact` times. If the
+function has not been called exactly `exact` times when
+[`tracker.verify()`][] is called, then [`tracker.verify()`][] will throw an
+error.
```

Comment:
This should explain how `withArgs` are compared to the actual arguments. I assume this implies deep strict-equal semantics, but some use cases would probably assume/prefer reference equality.

Reply:
Hey thanks! Just add this info on a new commit. Would you take a look at it?